apps/ledger-live-mobile/src/screens/Settings/Debug/Configuration/DebugEnv.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useMemo, useState, useCallback } from "react";
+import Clipboard from "@react-native-community/clipboard";
+import Config from "react-native-config";
+import {
+  EnvName,
+  setEnvUnsafe,
+  getAllEnvs,
+  getDefinition,
+} from "@ledgerhq/live-common/env";
+import { Flex, Button, Text, Tag, Alert } from "@ledgerhq/native-ui";
+import { TouchableOpacity } from "react-native-gesture-handler";
+import NavigationScrollView from "../../../../components/NavigationScrollView";
+import TextInput from "../../../../components/TextInput";
+import SectionSeparator from "../../../../components/SectionSeparator";
+
+export default function DebugEnv() {
+  const [value, setValue] = useState<string>("");
+  const [resetIndex, setResetIndex] = useState(0);
+  const [filter, setFilter] = useState<string>("");
+  const [status, setStatus] = useState<string>();
+  const [envDefinitions, setEnvDefinitions] = useState<{
+    [key: string]: string;
+  }>({});
+
+  const envs = useMemo(() => {
+    const envs = getAllEnvs();
+    const definitions = envDefinitions;
+
+    Object.keys(envs).forEach(key => {
+      if (!(key in definitions)) {
+        const maybeDefinition = getDefinition(key);
+        if (maybeDefinition) {
+          definitions[key] = maybeDefinition.desc;
```

Comment:
why `maybe` though ? There is a possibility that `getDefinition` returns something not `null`, but that is actually not a description ?

Reply:
🤔 I just wanted it to pass typecheck at that point, but indeed, the name doesn't make sense.
Feel free to commit if it bothers you